Resources/Prototypes/DeltaV/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -54,5 +54,12 @@
   id: VendingMachineBoozeUnlocked
   suffix: Unlocked
   components:
+  - type: VendingMachine
+    pack: BoozeOMatUnlockedInventory
+    offState: off
+    brokenState: broken
+    normalState: normal-unshaded
+    denyState: deny-unshaded
+    loopDeny: false
```

Comment:
why copy paste this

Reply:
> why copy paste this

I need to change the `pack` node. I thought everything under `-type` in yaml got overridden...?

Reply:
> > why copy paste this
> 
> I need to change the `pack` node. I thought everything under `-type` in yaml got overridden...?

pretty sure it isn't, but maybe it varies by type.

Reply:
it doesnt just change the 1 field